From b7900fbc0e929f2f340a44ed2881018a29a9a86f Mon Sep 17 00:00:00 2001
From: Josue Abarca <jmaslibre@gmail.com>
Date: Sun, 9 Aug 2020 15:48:15 -0600
Subject: [PATCH] Allow the command line verbose option to override the
 configuration file

Since the default configuration file has json_output set to true and
the json_output monopolizes stdout, superceding verbose, this change
allows the user to activate the verbose mode using the command line
parameter even when json_output is set to true in the configuration
file.
---
 src/main.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/main.c b/src/main.c
index 4bad685..2541cee 100644
--- a/src/main.c
+++ b/src/main.c
@@ -256,6 +256,8 @@ parse_cmdline(int argc, char *argv[])
         break;
       case 'v':
         my.verbose = TRUE;
+        my.json_output = FALSE;
+        my.quiet   = FALSE;
         break;
       case 'r':
         if(strmatch(optarg, "once")){
